__mocks__/fixtures.ts

```diff
@@ -692,6 +692,202 @@ export const visualizationObjects: IVisualization[] = [
             },
         },
     },
+    {
+        visualizationObject: {
+            content: {
+                buckets: [
+                    {
+                        localIdentifier: "measures",
+                        items: [
+                            {
+                                measure: {
+                                    localIdentifier: "fdd41e4ca6224cd2b5ecce15fdabf062",
+                                    format: "#,##0.00",
+                                    title: "Sum of Email Clicks",
+                                    definition: {
+                                        measureDefinition: {
+                                            computeRatio: true,
+                                            item: {
+                                                uri: "/gdc/md/yrungi0zwpoud7h1kmh6ldhp0vgkpi41/obj/15428",
```

Comment:
Please replace real project ids with the fake one

Reply:
Fixed!